apps/devhub/tests/test_views.py

```diff
@@ -1142,11 +1144,11 @@ def test_step1_submit(self):
     def test_read_dev_agreement_set(self):
         """Store current date when the user agrees with the user agreement."""
         response = self.client.get(reverse('devhub.submit.1'))
-        previous = response.context['user'].read_dev_agreement
+        assert response.context is None
```

Comment:
what is this testing? I don't think you need this assert for the purpose of this test.